Simplenote/Resources/AppStoreStrings.pot

```diff
@@ -63,12 +64,14 @@ msgstr ""
 #. translators: Keywords used in the App Store search engine to find the app.
 #. Delimit with an English comma between each keyword. Limit to 100 characters including spaces and commas.
 msgctxt "app_store_keywords"
-msgid "notes,text,planner,journal,diary,sync,to-do,cloud,notebook,simple,notepad,tag,todo,writing,memo"
+msgid "notes,text,planner,journal,diary,sync,to-do,cloud,notebook,simple,notepad,tag,todo,writing,memo
+"
```

Comment:
ditto here

```suggestion
msgid "notes,text,planner,journal,diary,sync,to-do,cloud,notebook,simple,notepad,tag,todo,writing,memo"
```

Reply:
I took a stub at fixing this issue here https://github.com/wordpress-mobile/release-toolkit/pull/297.